src/NuGet/NugetProjectPackUtil.csproj

```diff
@@ -0,0 +1,97 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Rename for correct casing.

Reply:
What is the correct casing: nET?

Reply:
File name, not this particular line. I just can't make the comment on the file.

Reply:
Oh for FFS that didn't get fixed? I tried threed different times to fix the case on that file.